src/inverseRobot.js

```diff
@@ -24,7 +24,23 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const goodRobot = {};
+  let isWrongRobot = false;
+
+  for (const key in robot) {
+    if (goodRobot.hasOwnProperty(robot[key])) {
+      isWrongRobot = true;
+      break;
```

Comment:
instead of using unnecessary variables you can return null here
```suggestion
      return null
```